### PR TITLE
Reduce noise in logs

### PR DESCRIPTION
**Pull request by uthark**
Change log level from info to debug for *Handle...* messages. They are useless for production usage and just generate a lot of noise in logs.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by uthark** [comment]
@mogren Please take a look.

File: pkg/eniconfig/eniconfig.go [hunk 1]
```diff
@@ -131,8 +131,10 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 
 			h.controller.eniLock.Lock()
 			defer h.controller.eniLock.Unlock()
-			h.controller.myENI = val
-			log.Infof("Setting myENI to: %s", val)
+			if h.controller.myENI != val {
```
Comment (mogren) [inline pkg/eniconfig/eniconfig.go]: If you're checking this variable, you might as well move the lock inside the if-statement as well.

File: pkg/eniconfig/eniconfig.go [hunk 2]
```diff
@@ -111,14 +111,14 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 
 		curENIConfig := o.DeepCopy()
 
-		log.Infof("Handle ENIConfig Add/Update: %s, %v, %s", eniConfigName, curENIConfig.Spec.SecurityGroups, curENIConfig.Spec.Subnet)
+		log.Debugf("Handle ENIConfig Add/Update: %s, %v, %s", eniConfigName, curENIConfig.Spec.SecurityGroups, curENIConfig.Spec.Subnet)
```
Comment (mogren) [inline pkg/eniconfig/eniconfig.go]: I agree that Info is high for this.

File: pkg/eniconfig/eniconfig.go (hunk 1 above)
Comment (uthark) [reply pkg/eniconfig/eniconfig.go]: Ah, nice catch. Fixed.

**Pull request merged**